gui/src/utils/index.ts

```diff
@@ -67,7 +87,7 @@ export const getDateTime = (value: string | null | undefined, tz?: string): Date
         return null;
     }
     try {
-        return (tz && tz !== "Etc/Unknown") ? utcToZonedTime(value, tz) : new Date(value);
+        return tz && tz !== "Etc/Unknown" ? utcToZonedTime(value, tz) : new Date(value);
```

Comment:
I do enjoy the parenthesis that in my opinion make the code more readable.
No big deal...

Reply:
but the formatter removes them!